openedx2zim/scraper.py

```diff
@@ -859,16 +855,23 @@ def run(self):
                 self.fname or f"{self.name.replace(' ', '-')}_{{period}}.zim"
             ).format(period=datetime.datetime.now().strftime("%Y-%m"))
             logger.info("building ZIM file")
-            self.update_zim_info()
-            logger.debug(self.zim_info.to_zimwriterfs_args())
+            zim_info = self.get_zim_info()
             if not self.output_dir.exists():
                 self.output_dir.mkdir(parents=True)
             make_zim_file(
-                self.build_dir,
-                self.output_dir,
-                self.fname,
-                self.zim_info,
-                withoutFTIndex=True if self.no_fulltext_index else False,
+                build_dir=self.build_dir,
+                fpath=self.output_dir.joinpath(self.fname),
+                name=self.name,
+                main_page=zim_info["homepage"],
+                favicon="favicon.png",
+                title=zim_info["title"],
+                description=zim_info["description"],
+                language="eng",
+                creator=zim_info["creator"],
+                publisher=self.publisher,
+                tags=self.tags + ["_category:openedx", "openedx"],
```

Comment:
We have no openedx category. Setting it will make the zims invisible in apple readers as the library is based on the category. Use other.

Reply:
Okay. Did that.